apps/andi/lib/andi/harvest/harvester.ex

```diff
@@ -48,13 +66,43 @@ defmodule Andi.Harvest.Harvester do
 
   def dataset_update(datasets) do
     Enum.each(datasets, fn dataset ->
-      Brook.Event.send(instance_name(), dataset_update(), :andi, dataset)
+      case Organizations.get_harvested_dataset(dataset.id) do
```

Comment:
wait, what's the difference between this and the function below?

Reply:
dataset_update sends the update dataset event with the smart city dataset and harvested dataset update sends the harvest end event and updated the harvested dataset table.